tests/index.html

```diff
@@ -1,24 +1,23 @@
 <!DOCTYPE html>
 <html>
   <head>
-	<script src="../node_modules/qunitjs/qunit/qunit.js"></script>
+    <script src="../node_modules/qunitjs/qunit/qunit.js"></script>
```

Comment:
Is there any reason the format was changed?

Reply:
Just for consistency. It was a mix of tabs and spaces, I converted all to spaces.